package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "azure-devops-workitem-link",
+  "version": "1.0.0",
+  "description": "<div align=\"center\">   <a href=\"https://github.com/adnanjaw/azure-devops-workitem-link\">     <img src=\"src/assets/favicon/logo-tran.png\" alt=\"Logo\" width=\"190\" height=\"190\">   </a> <div align=\"center\">",
```

```suggestion
  "description": "Adds dynamic and static links to Azure DevOps work items.",
```

Better to remove HTML and have useful a description (better for [indexing](https://nodesource.com/blog/the-basics-of-package-json#thedescriptionproperty))🙃

---

Good catch. That happens when you generate **package.json** using phpStorm. Phpstorm uses a readme file to create the package.json